app/src/main/java/org/p2p/wallet/svl/interactor/ReceiveViaLinkInteractor.kt

```diff
@@ -22,8 +23,15 @@ class ReceiveViaLinkInteractor(
     private val tokenKeyProvider: TokenKeyProvider
 ) {
 
+    private class ReceiveViaLinkError(
+        override val message: String,
+        override val cause: Throwable? = null
+    ) : Throwable()
```

Comment:
move to a separate file

Reply:
Not used anywhere than here, are you sure?

Reply:
Not at all, it's for Sentry and Firebase for specific details in issue list